src/page-multi-facility/SystemSummary.tsx

```diff
@@ -0,0 +1,129 @@
+import React, { useContext } from "react";
+import styled from "styled-components";
+
+import { FetchedFacilities } from "../constants";
+import Colors from "../design-system/Colors";
+import Loading from "../design-system/Loading";
+import Metric from "../design-system/Metric";
+import {
+  getTotalPopulation,
+  totalIncarceratedConfirmedCases,
+} from "../impact-dashboard/EpidemicModelContext";
+import { isRtData, RtData, RtRecord } from "../infection-model/rt";
+import * as rtStats from "../page-response-impact/rtStatistics";
+import { FacilityContext } from "./FacilityContext";
+import { Facilities, Facility } from "./types";
+
+const SystemSummaryContainer = styled.div`
+  border-bottom: 1px solid ${Colors.opacityGray};
+  margin-bottom: 15px;
+  min-height: 230px;
+`;
+
+const SectionHeader = styled.div`
+  color: ${Colors.forest};
+  font-family: "Libre Baskerville", serif;
+  font-size: 24px;
+  letter-spacing: -0.06em;
+  line-height: 24px;
+  text-align: left;
+  padding-bottom: 15px;
+  font-weight: normal;
+`;
+
+const MetricsContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  margin: 15px;
+`;
+
+interface Props {
+  facilities: FetchedFacilities;
+}
+
+const SystemSummary: React.FC<Props> = ({ facilities }) => {
+  const numFacilities = facilities.data.length;
+
+  const { rtData } = useContext(FacilityContext);
+  const rtDataValues: (RtData | null)[] = Object.values({ ...rtData });
+
+  const facilitiesRtRecords: RtRecord[][] = rtDataValues
+    .filter(isRtData)
+    .map((rtData: RtData) => rtData.Rt);
+
+  const metrics = {
+    incarceratedPopulation: {
+      value: 0,
+      label: "Total resident population",
+    },
+    staffPopulation: {
+      value: 0,
+      label: "Total staff population",
+    },
+    facilitiesWithCases: {
+      get value() {
+        return `${this.sum} of ${numFacilities}`;
+      },
+      label: "Facilities with confirmed cases",
+      sum: 0,
+    },
+    incarceratedCases: {
+      value: 0,
+      label: "Total resident cases",
+    },
+    staffCases: {
+      value: 0,
+      label: "Total staff cases",
+    },
+    facilitiesWithRtLessThan1: {
+      value: `${rtStats.numFacilitiesWithRtLessThan1(
+        facilitiesRtRecords,
+      )} of ${numFacilities}`,
```

Comment:
So just to be sure, if I have 8 facilities and 4 have Rt less than 1, and 2 have Rt greater than 1, and 2 have no Rt value at all yet, then we would say "4 of 8", right? We wouldn't say "4 of 6" or "6 of 8"?

Reply:
Great question, I need to check on that.

Reply:
Confirmed it is "6 of 8" We want to include all facilities in this total, even if they don't have an rT value.

Reply:
I'm willing to accept that, as long as we are aware that 2 of those 6 are facilities for which we do not yet have enough information to classify.

Reply:
Yes! One way to potentially help clarify this to users is updating the copy to say "facilities with a classifiable slowing rate of spread" to demonstrate that the number is representing only the facilities where an r(t) can be calculated _and_ have r(t) < 1.

Reply:
@sychang if we made that copy change, then that would mean we want the calculation to be the "4 of 8" scenario I outlined above, not the "6 of 8", since there would only be 4 that have a calculable R(t) which is less than 1. If we are doing the 6 of 8, then our text should represent that the number is representing facilities where an R(t) is less than 1 _or_ where R(t) has not been calculated yet.

Does that make sense? What do we want for this?

Reply:
4 of 8 in this example!